frontend/javascripts/oxalis/view/components/setting_input_views.tsx

```diff
@@ -378,6 +378,7 @@ type UserBoundingBoxInputProps = {
   disabled: boolean;
   isLockedByOwner: boolean;
   isOwner: boolean;
+  id?: string | undefined;
```

Comment:
Hmmm, my thought was to not enforce an id for each bounding box input element created and thus made it optional. I wanted to avoid in case the component is used elsewhere in the code, it must also need a unique id. 
-> But in the current version of the code base `UserBoundingBoxInput` is only used in the bbox tab in the table (where the id is needed).
=> I removed the optional from `id`. In case `UserBoundingBoxInput` is used elsewhere and an id is not necessary, it can be made optional again :)